ValidicCSharp/Request/UrlBuilder.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace ValidicCSharp.Request
+{
+    using System.Linq;
+
+    public class UrlBuilder
+    {
+        public static string FromCommand(Command command)
```

Comment:
Would like to see more unit tests for this class that fail instead of just happy path